### ISSUE TITLE
Boxee continues to reboot itself

### ISSUE THREAD

**Issue by teddawson** [boxeehacks/boxeehack#470]
I installed Boxee+ three days ago, the first day I saw no problems however, over the past two days I have not been able to use it because it continually reboots itself about every 3 -5 minutes. Is there anything I can try to resolve this issue or can I reset the unit back to its original config?

**Comment by Kepners** [comment]
Boxee Restarting every 60second "error script fail: Boxeehack_server.py" is popping up. It just started a few days ago, but would run without issue. However today boxee wont stay on more than 60second before restarting popping up that error.

Going to have to remove Boxeehack until this is resolved.

I await to hear feed back.

Keps

**Comment by Kepners** [comment]
I have just reverted back what seems is the original setup and its still the same. however there is no POPUP window reporting the error.

**Comment by teddawson** [comment]
Tes, I reverted back to the original Boxee setup yesterday and things have been as usual before Boxee+. However, the original setup was never the best. long loading times when streaming movies via NAVI-X etc. Will remain in holding pattern until I hear something about possible fix from Boxeehack.

**Comment by teddawson** [comment]
Should have been "Yes" not Tes, sorry.

**Comment by blp81** [comment]
Since updating latest version of boxee hacks, boxee restarts itself, and I'm not able to get into any options or settings at all. Can anyone tell me if I can somehow reset boxee box to original settings so that it's useable again? Thanks

**Comment by Kepners** [comment]
Hi,
I have had to revert back to the original firmware then update yesterday evening... I have since found doing this that I believe the error is not with the boxes or the boxes+ hack. because! The same error occurred this morning with the standard boxee box software installed. It kept rebooting after about 60seconds. I have since had to reset my boxes settings. This has solved the problem again. I believe the issue is to do with the fan art programs are creating all this database stuff and its corrupting the boxes box.
I am adding one NAS at a time to see which one is causing the corruption - as I have used two different types of metadata software programs.

**Comment by blp81** [comment]
Is there anyway I can reset the boxee box settings with the condition it's in? I can't use the boxee remote to control box as it's restarting itself very quickly, and doesn't allow me to get into settings or anything. Thanks again.

**Comment by Kepners** [comment]
You can get to its DOS mode...(by using the button on the boxee and holding it for over 6 seconds upon startup) but it doesnt allow you to keep your current corrupted settings. it renews them all or can return to manufacturers settings. I have had to do both in the last 24hours, but the boxee is back! and all hacked up!

**Comment by blp81** [comment]
Hey thanks alot Kepners, that worked. Restored to factory settings and it's working good.

**Comment by Kepners** [comment]
Yup No worries and you'll Boxee+ will work better! it found everything this time for me. Glad it sorted itself out for you.

**Comment by blp81** [comment]
Hi again. now that i've reset boxee, netflix app isn't working. I hit the netflix app and the screen goes black for a moment and then goes back to where I was. I've re-installed boxee hacks but when I try netflix there it does the same thing. I re-set boxee again also but still no go with the netflix. Any ideas for a fix for this problem? Thanks

**Comment by bradley4681** [comment]
Ok so I have 2 boxes and I recently installed boxee+ on one of them. after a few days they both started acting weird and rebooting, BOTH! 
The following applies to symptoms experienced on both boxee boxes randomly in the last week and a half, but i have described as a single box for the ease of reading. 

I would restore to the factory restore and then reboot, sometimes I could set up the system because it would say its not connected to the internet, or it would only connect to a wired network. sometimes it would connect and let me login and as soon at the home screen would load it would restart. now when i say restart i mean the interface would restart, the spinning circle and then back to the home screen. sometimes i had enough time to change the selected icon. if i left it to continue on i would usually come back to an unresponsive system and a black screen. 

At this point i could reset it again and sometimes experience nothing for an hour, or a day, or maybe again right away and all random on both boxes. Nothing in my home, my ISP, my setups had any changes made except the one boxee box with boxee+, which was never reinstalled after the first firmware restore. It seems from the forums that quite a few people lately have been experiencing issues with reboots/resetting and/or connecting to the internet.

see post #24 here
http://forums.boxee.tv/showthread.php?t=65161&page=3

That kind of thing worries me with the end of support coming shortly.
